cmd/celestia/admin.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"strings"
+
+	"github.com/filecoin-project/go-jsonrpc/auth"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	adminCmd.AddCommand(nodeInfoCmd, logCmd, verifyCmd, authCmd)
+	rootCmd.AddCommand(adminCmd)
+}
+
+var adminCmd = &cobra.Command{
+	Use:   "admin [command]",
```

Comment:
It would be nice to have the name the same as modules name.

Reply:
done